site/templates/meet.json.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * @var Kirby\Cms\Pages $upcoming
+ * @var Kirby\Cms\Pages $past
```

Comment:
`Cleanup`: Fully-qualified class names improve compatibility with IDE tools:

```suggestion
 * @var \Kirby\Cms\Pages $upcoming
 * @var \Kirby\Cms\Pages $past
```

Reply:
If I do that, my IDE tells me to replace qualifiers with an import...